config/environments/production.rb

```diff
@@ -49,9 +61,9 @@
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
   # config.force_ssl = true
 
-  # Use the lowest log level to ensure availability of diagnostic information
-  # when problems arise.
-  config.log_level = :debug
+  # Include generic and useful information about system operation, but avoid logging too much
+  # information to avoid inadvertent exposure of personally identifiable information (PII).
+  config.log_level = :info
```

Comment:
Why is this being raised from debug to info?

Reply:
the comment right before this line explains the reasoning behind the raising of the level
And also, in Rails 7.1 (which will be done after this upgrade), the default `log_level` will be set to `:info` in production, and hence it should be good practice to start it from now.

https://guides.rubyonrails.org/v7.0.8.4/debugging_rails_applications.html

Reply:
Ok, but will need to monitor the impact on the amount of logs we keep.  
Previously we had an issue with lograge generating a lot of logs which cased our AWS bill to spike.

Reply:
Roger that